BUILD.md

```diff
@@ -170,32 +170,43 @@ Until then, we advise Windows developers to use Visual Studio 2019.
 
 ## Prerequisites
 
-To build this package, you will need Python (>= 3.7),
-[Conan][] (>= 1.55), and [CMake][] (>= 3.16).
-
 > **Warning**
-> The commands in this document are not meant to be blindly copied and pasted.
+> The commands in this document are guidelines which cannot always be successfully copy and pasted.
 > This document is written for multiple audiences,
 > meaning that your particular circumstances may require some commands and not
 > others.
-> You should never run any commands without understanding what they do
-> and why you are running them.
+> It is recommended to fully understand what the following commands do before running them.
 >
 > These instructions assume a basic familiarity with Conan and CMake.
 > If you are unfamiliar with Conan,
 > then please read the [crash course](#a-crash-course-in-cmake-and-conan)
 > at the beginning of this document,
 > or the official [Getting Started][3] walkthrough.
 
+### All Platforms
+
+To build this package, you will need [Python][] (>= 3.7),
+[Conan][] (>= 1.52), and [CMake][] (>= 3.16).
```

Comment:
I do not believe Conan 1.52 is sufficient any more. We should recommend the latest anyway given all the recent changes (1.58).

Reply:
Fixed.